underdocs-renderer/src/main/kotlin/underdocs/renderer/output/html/render/page/PageRenderer.kt

```diff
@@ -57,8 +60,26 @@ class PageRenderer(private val linker: Linker,
                             .withHref(linker.siteLinkBetween(visitable, "_static/code.css"))
             ),
             body(
+                    script()
+                            .with(UnescapedText("""
+                                (function () {
+                                  if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+                                    document.body.classList.add('night');
+                                  }
+                                })();
+                            """.trimIndent())),
                     main(
-                            crumbRenderer.render(visitable),
+                            div(
+                                    crumbRenderer.render(visitable),
+                                    div(
+                                            span("Night Mode"),
+                                            input()
+                                                    .withType("checkbox")
+                                                    .withId("switch"),
```

Comment:
In the future, we might have other switches on the page. Thus, a more distinctive identifier would be advisable for this toggle. Can you please change the identifier to a more appropriate one?

Reply:
You're right, renamed.